wiki.py

```diff
@@ -658,15 +658,76 @@ def single_project_info(self, project_id, sv_name):
             elif v.get("title"):
                 pages.append(self._make_year_title(v["title"]))
         logging.warning(
-            "Don't forget to manually add '{id} - {name}' to the following "
-            "pages: {pages}".format(
-                id=project_id, name=sv_name, pages='\n* '.join(pages)
+            "Don't forget to manually add '{number} - {name}' to the "
+            "following pages: {pages}".format(
+                number=number, name=sv_name, pages='\n* '.join(pages)
             )
         )
 
     def log_report(self):
-        """Log a list of the pages that were modified.
-        """
+        """Log a list of the pages that were modified."""
         logging.info("These pages were modified:")
         for page in self._touched_pages:
             logging.info(page.title())
+
+    def update_project_name_template(self):
```

Comment:
`update_project_name_templates` would be a better name to make it clear that both are being affected